partiql-ast/src/ast.rs

```diff
@@ -319,6 +320,8 @@ pub enum ExprKind {
     Call(CallAst),
     CallAgg(CallAggAst),
 
+    Type(TypeAst),
```

Comment:
nit: could also add some documentation for this enum variant since it's public

Reply:
Makes sense—added.